tools/manifest-generator/manifest-generator.go

```diff
@@ -0,0 +1,52 @@
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package main
+
+import (
+	"text/template"
+	"os"
+	"flag"
+	"github.com/golang/glog"
+)
+
+type data struct {
+	DockerRepo string
+	DockerTag  string
+	Verbosity  int
+}
+
+func main () {
+	dockerRepo := flag.String("docker-repo", "", "")
+	dockertag := flag.String("docker-tag", "", "")
+	templFile := flag.String("template", "", "")
```

Comment:
would `-o=` be ok here? Potentially confusing if `-templFile` names the input or output file. Plus, since it's abbreviated users will forget and type `-template=`, imo